.vscode/cspell.json

```diff
@@ -58,6 +54,7 @@
     "azsdk",
     "blazor",
     "contoso",
+    "deduplication",
```

Comment:
Is this needed for ACR anon access?

Reply:
Yeah, these changes are needed to keeping the tooling happy. CI builds now have additional checks and I brought the minimum changes in to pass the CI checks (cspell and sparse-checkout fixes).